### ISSUE TITLE
Types broken in 3.0.3

### ISSUE THREAD

**Issue by qtiki** [pixijs/pixi-sound#114]
Type definitions are broken in 3.0.3 for `SoundUtils.resolveUrl`.

The implementation of the function is properly typed to use `PIXI.LoaderResource`: https://github.com/pixijs/pixi-sound/blob/26303661a88f7e4f908307dc6552e831cf0dd86a/src/utils/resolveUrl.ts#L23

However the distributed `pixi-sound.d.ts` contains a broken reference to `PIXI.loaders.Resource`: https://github.com/pixijs/pixi-sound/blob/26303661a88f7e4f908307dc6552e831cf0dd86a/pixi-sound.d.ts#L200

This results in a compilation error:
```
/node_modules/pixi-sound/pixi-sound.d.ts:200:57
  TS2694: Namespace 'PIXI.loaders' has no exported member 'Resource'.
```

I'm guessing this bug has previously been hidden since we've been using 3.0.3 for a long time. It's now surfaced very likely due to the new Pixi.js release.

**Comment by bigtimebuddy** [comment]
This makes sense actually because we removed deprecations from the PixiJS v5 typings. This exposed this bug. I’ll push out a release ASAP with this fix. Thanks for reporting.

**Comment by bigtimebuddy** [comment]
Released v3.0.4
